src/tests/Microsoft.PowerFx.Core.Tests.Shared/IRTests/IRPrettyPrinter.cs

```diff
@@ -190,6 +191,10 @@ public override RetVal Visit(ResolvedObjectNode node, Context context)
             {
                 _sb.Append(ee.EntityName);
             }
+            else if (node.Value is FormulaValue fv)
+            {
+                _sb.Append(fv.ToExpression());
```

Comment:
Do you want compact or verbose?

Reply:
I think this is fine, its just to display "smuggled" node. We can change in future if we need compact vs verbose setting.